graalpy-embedding-demo/pom.xml

```diff
@@ -0,0 +1,170 @@
+ data (collectively the "Software"), free of charge and under any and all
+ copyright rights in the Software, and any and all patent rights owned or
+ freely licensable by each licensor hereunder covering either (i) the
+ unmodified Software as contributed to or provided by such licensor, or (ii)
+ the Larger Works (as defined below), to deal in both
+
+ (a) the Software, and
+
+ (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ one is included with the Software each a "Larger Work" to which the Software
+ is contributed by such licensors),
+
+ without restriction, including without limitation the rights to copy, create
+ derivative works of, display, perform, and distribute the Software and make,
+ use, sell, offer for sale, import, export, have made, and have sold the
+ Software and the Larger Work(s), and to sublicense the foregoing rights on
+ either these or other terms.
+
+ This license is subject to the following condition:
+
+ The above copyright notice and either this complete permission notice or at a
+ minimum a reference to the UPL must be included in all copies or substantial
+ portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.example</groupId>
+  <artifactId>embedded_graalpy</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+    <maven.jar.plugin.version>3.3.0</maven.jar.plugin.version>
+    <native.image.maven.plugin.version>0.9.28</native.image.maven.plugin.version>
+    <!-- Select the GraalPy version to use. -->
+    <graalpy.version>23.1.2</graalpy.version>
+    <!-- Specify the main class -->
+    <main.class>com.oracle.example.graalpy.Main</main.class>
+  </properties>
+
+  <dependencies>
+    <!-- The polyglot dependency is always required  -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>polyglot</artifactId>
+      <version>${graalpy.version}</version>
+      <type>jar</type>
+    </dependency>
+
+    <!-- The GraalPy dependency -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>python</artifactId>
+      <version>${graalpy.version}</version>
+      <scope>runtime</scope>
```

Comment:
The `pom` type is a requirement for language or tool dependencies, not `runtime`.

Reply:
From https://www.graalvm.org/latest/reference-manual/embed-languages/#dependency-setup

Reply:
That's odd, because this works without the `pom` type and doesn't seem to be included in the maven archetype.
Could you confirm @timfel ?

Reply:
Maybe there was an update since the last time I tested.

Reply:
The `pom` type should be there.